hoover/contrib/twofactor/views.py

```diff
@@ -58,3 +60,9 @@ def invitation(request, code):
         'bad_password': bad_password,
         'bad_token': bad_token,
     })
+
+urlpatterns = [
```

Comment:
Urls in `views.py`?
Can't wait to have to search for these and `grep` my way to madness.

Could you make a `hoover.contrib.twofactor.urls` module and use [`include('hoover.contrib.twofactor.urls')`](https://docs.djangoproject.com/en/1.10/topics/http/urls/#including-other-urlconfs)?

That way, other people in this project only have to follow the Django convention to stay sane.